tensorflow_datasets/translate/wmt.py

```diff
@@ -778,7 +777,8 @@ def _get_local_paths(ds, extract_dirs):
           continue
         # TODO(adarob): Add subset feature.
         # ex["subset"] = subset
-        yield ex
+        key = '{}/{}'.format(ss_name, ex)
```

Comment:
You shouldn't use the full example as key, as it won't be deterministically generated.
Instead, as explained previously, you need to create a unique key for each subgenerator (`_parse_tvs`, ...):

```
for ss_name in split_subsets:
  ...
  for sub_key, ex in sub_generator(*files):
    ...
    key = '{}/{}'.format(ss_name, sub_key)
    yield key, ex
```